composer.json

```diff
@@ -13,19 +13,19 @@
     }
   ],
   "require": {
-    "php": "^8.1",
+    "php": "^8.2",
     "guzzlehttp/guzzle": "^7.5",
-    "illuminate/database": "^10.0",
-    "illuminate/encryption": "^10.0",
-    "illuminate/hashing": "^10.0",
-    "illuminate/http": "^10.0",
-    "illuminate/queue": "^10.0",
-    "illuminate/routing": "^10.0",
-    "illuminate/translation": "^10.0",
-    "illuminate/validation": "^10.0",
-    "laravel-zero/framework": "^10.2",
-    "laravel/sanctum": "^3.3",
-    "nunomaduro/termwind": "^1.15.1",
+    "illuminate/database": "^11.5",
+    "illuminate/encryption": "^11.5",
+    "illuminate/hashing": "^11.5",
+    "illuminate/http": "^11.5",
+    "illuminate/queue": "^11.5",
+    "illuminate/routing": "^11.5",
+    "illuminate/translation": "^11.5",
+    "illuminate/validation": "^11.5",
+    "laravel-zero/framework": "^11.0",
+    "laravel/sanctum": "^4.0",
+    "nunomaduro/termwind": "^2.0",
```

Comment:
this is included with laravel-zero/framework and does not need to be a direct dependency

```suggestion
```